aurora/core.py

```diff
@@ -1096,7 +1096,8 @@ def calc_Zeff(self):
         Z = np.arange(Zmax + 1)
         self.delta_Zeff = nz * (Z * (Z - 1))[None, :, None]  # for each charge state
         self.delta_Zeff /= self.ne.T[:, None, :]
-
+        return self.delta_Zeff
```

Comment:
@abhatt03 the `return` statement was intentionally avoided here, because `delta_Zeff` is anyway cached in `self`. This means that you can anyway access the result of your `aurora_sim` instantiation (say, called `asim`), as `asim.delta_Zeff`.

Reply:
@fsciortino : I am attaching three screenshots to show the reason why I made the changes. Forgive me if the screenshots are too green. In the Aurora manual, as I read before starting to work with Aurora, the function to obtain the Zeff values is calc_Zeff() in core.py (Zeff values obtained with asim.calc_Zeff()). 

I am attaching the screenshots, where, two of the screenshots show error received with asim.delta_Zeff() (it may be helpful to mention this is the quantity and not calc_Zeff() in the manual if a return statement is not desired) and asim.calc_Zeff(), when the return statement is commented. The third one shows what happens when the return statement is uncommented. 

![cal_zeff_error_received_Aurora1](https://user-images.githubusercontent.com/92543947/166122326-0dad32d9-1075-4eff-97fa-d0b312413475.png)

![cal_zeff_error_received_Aurora2](https://user-images.githubusercontent.com/92543947/166122328-d4975c12-819e-4639-b630-95a55d8d3492.png)

![cal_zeff_error_corrected_received_Aurora1](https://user-images.githubusercontent.com/92543947/166122332-3d470962-e3c4-404a-a6d4-121642717c52.png)

As these errors are what I have received, whatever changes I have mentioned until now are simply stated/intended as a suggestion.